modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -263,6 +268,9 @@ public class IgniteImpl implements Ignite {
 
     private final ScheduledExecutorService raftExecutorService;
 
+    /** CompletableFuture of {@link ClusterRestConfiguration} that will be resolved after the cluster initialization. */
```

Comment:
It is not a `CompletableFuture`, it probably contains the future. Looks like this comment needs to be refined.